streams/src/main/java/org/apache/kafka/streams/kstream/internals/graph/ProcessorGraphNode.java

```diff
@@ -28,6 +28,13 @@
 
     private final ProcessorParameters<K, V> processorParameters;
 
+    public ProcessorGraphNode(final ProcessorParameters<K, V> processorParameters) {
```

Comment:
Added this constructor so that we don't have to keep variables for all the processor names just to pass them in to both the node and processor.